cmd/sonobuoy/app/retrieve.go

```diff
@@ -77,7 +75,7 @@ func retrieveResults(cmd *cobra.Command, args []string) {
 	eg := &errgroup.Group{}
 	eg.Go(func() error { return <-ec })
 	eg.Go(func() error {
-		filesCreated, err := client.UntarAll(reader, outDir, prefix)
+		filesCreated, err := client.UntarAll(reader, outDir, "")
```

Comment:
Will this change work as before? If not, will that change any docs ?

Reply:
This will make the result the same as before as if we still relied on `tar` binary provided by OS / distro